scripts/optimize.py

```diff
@@ -49,10 +61,13 @@
     # create model from energy system (this is just oemof.solph)
     m = Model(es)
 
+    # Add an emission constraint
+    constraints.emission_limit(m, limit=get_emission_limit())
```

Comment:
It would save time to retrieve the emission limit before building the model. get_emission_limit is much faster than building the model. If it fails, it is better to notice early.